packages/cli-kit/src/private/node/ui/components/Tasks.tsx

```diff
@@ -65,9 +68,13 @@ function Tasks<TContext>({
   silent = isUnitTest(),
   onComplete = noop,
   abortSignal,
+  noColor,
 }: React.PropsWithChildren<TasksProps<TContext>>) {
   const {twoThirds} = useLayout()
-  const loadingBar = new Array(twoThirds).fill(loadingBarChar).join('')
+  let loadingBar = new Array(twoThirds).fill(loadingBarChar).join('')
+  if (noColor ?? !shouldDisplayColors()) {
+    loadingBar = hillString.repeat(Math.ceil(twoThirds / hillString.length))
```

Comment:
Rounding up here so we have a bit of extra insurance at the end (which `TextAnimation` will truncate), rather than overtruncating immediately.